Resources/Prototypes/_Goobstation/Research/tags.yml

```
@@ -0,0 +1,104 @@
+
+- type: Tag
+  id: RipleyMkII
+
+- type: Tag
+  id: Clarke
+
+- type: Tag
+  id: Durand
+
+- type: Tag
+  id: Gygax
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)